.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: node_js
```

Comment:
use github action?

Reply:
It's just something to think about. Do you think it would be a good idea?

Reply:
Year, GitHub action is excellent, but change to GitHub is not an emergency.